packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -70,9 +70,18 @@ export const messages = {
 
   force_apex_test_run_text: 'SFDX: Invoke Apex Tests...',
   force_apex_test_run_all_test_label: 'All tests',
-  force_apex_test_run_all_tests_desription_text:
+  force_apex_test_run_all_tests_description_text:
     'Runs all tests in the current project',
 
+  force_apex_test_run_codeAction_all_tests_text:
+    'SFDX: Invoke All Apex Tests in Class',
```

Comment:
Is there a reason to not use "Run" instead of "Invoke" in these commands? Seems to me like people are more likely to search for "run" than "invoke."

Reply:
Was just being consistent w/ existing "SFDX: Invoke Apex Tests...".

Let me know if you want me to change all references.

Reply:
Ah, ok. I don't want to change what people are already using. Carry on...